tests/net/6lo/src/main.c

```diff
@@ -947,18 +934,18 @@ void main(void)
 	net_6lo_set_context(net_if_get_default(), &ctx2);
 #endif
 
-	for (count = 0, pass = 0; count < ARRAY_SIZE(tests); count++) {
+	for (count = 0; count < ARRAY_SIZE(tests); count++) {
 		TC_START(tests[count].name);
 
-		if (test_6lo(tests[count].data)) {
-			TC_END(FAIL, "failed\n");
-		} else {
-			TC_END(PASS, "passed\n");
-			pass++;
-		}
+		test_6lo(tests[count].data);
 	}
-
 	net_pkt_print();
+}
 
-	TC_END_REPORT(((pass != ARRAY_SIZE(tests)) ? TC_FAIL : TC_PASS));
+/*test case main entry*/
+void test_main(void)
+{
+	ztest_test_suite(test_6lo,
+			 ztest_unit_test(test_loop));
```

Comment:
why breaking here? it seems like all can fit in one line.

Reply:
Fixed it